packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -244,7 +280,9 @@ export class TokenRatesController extends BaseController<
    * @param _chainId - The current chain ID.
    */
   set chainId(_chainId: Hex) {
-    !this.disabled && safelyExecute(() => this.updateExchangeRates());
```

Comment:
The `this.disabled` check is intentionally removed here because it was redundant (it's checked at the start of `updateExchangerates`)